main.py

```diff
@@ -1,10 +1,40 @@
+import json
 import init_django_orm  # noqa: F401
-
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file:
+        players_data = json.load(file)
```

Comment:
Consider adding error handling for the JSON loading process. If the file is not found or the JSON is malformed, it could cause the program to crash.